items/items.js

```diff
@@ -89,39 +91,61 @@ class Item {
   }
 
   /**
-   * The type of the Item: the Simple (without package) name of the Java Item type, such as 'Switch'.
+   * Type of Item: the Simple (without package) name of the Java Item type, such as 'Switch'.
    * @type {string}
    */
   get type () {
     return this.rawItem.getClass().getSimpleName();
   }
 
   /**
-   * The name of the Item.
+   * Name of Item
    * @type {string}
    */
   get name () {
     return this.rawItem.getName();
   }
 
   /**
-   * The label attached to the Item.
+   * Label attached to Item
    * @type {string}
    */
   get label () {
     return this.rawItem.getLabel();
   }
 
   /**
-   * The state of the Item.
+   * String representation of the Item state.
    * @type {string}
    */
   get state () {
     return this.rawState.toString();
   }
 
   /**
-   * The raw state of the Item, as a Java {@link https://www.openhab.org/javadoc/latest/org/openhab/core/types/state State object}.
+   * Numeric representation of Item state, or `null` if state is not numeric
+   * @type {number|null}
+   */
+  get numericState () {
+    const numericState = parseFloat(this.rawState.toString());
+    return isNaN(numericState) ? null : numericState;
```

Comment:
Should we return null, or NaN ?  Since this is a native javascript return type, i'm a little on the fence.

Reply:
I have no strong opinion here. 
If we return NaN, we should change the unreleased `HistoricItem` as well.

What kind of speaks against returning NaN here is, that quantityState returns null if not possible so returning null here as well would make it more inline.

Reply:
I think null works just as well then.